tests/e2e-playwright/tests/jupyterlab.py

```diff
@@ -63,16 +63,28 @@ def test_jupyterlab(
 
     # Wait until iframe is shown and create new notebook with print statement
     page.frame_locator(".qx-main-dark").get_by_role(
-        "button", name="New Launcher (Ctrl+Shift+L)"
+        "button", name="New Launcher"
     ).click(timeout=600000)
-    page.frame_locator(".qx-main-dark").locator(".jp-LauncherCard-icon").first.click()
-    _jupyterlab_ui = (
-        page.frame_locator(".qx-main-dark")
-        .get_by_label("Untitled.ipynb")
-        .get_by_role("textbox")
-    )
-    _jupyterlab_ui.fill("print('test')")
-    _jupyterlab_ui.press("Shift+Enter")
+    if "jupyter-octave-python-math" in service_key or "jupyter-math" in service_key:
```

Comment:
At some point I would prefer not having service-specific values/info in here. 
Ideally every service can register its own e2e test-snippets
If we continue with this if/elif testing apporach we will go back to the e2e's we are trying to improve

Reply:
If we start to see issues for sure we might improve it, for now, it is fine like this, these tests are already running from this branch in the Batman slides :) 

As there are inconsistencies in services across each deployment (services itself + their versions) I just modified the test so it is more flexible and tests the basic functionality across multiple jupyterlab services that we provide. You can check which ones in our e2e repo: https://git.speag.com/oSparc/e2e-backend